splot/_viz_value_by_alpha.py

```diff
@@ -0,0 +1,36 @@
+import geopandas as gpd
+from shapely.geometry import Polygon
+import matplotlib
+
+"""
+Creating Value by Alpha maps
+
+"""
+
+__author__ = ("Stefanie Lumnitz <stefanie.lumitz@gmail.com>")
+
+
+def value_by_alpha(x, alphay, gdf):
+    
+
+
+
+def value_by_alpha(x='attribute', y='alpha_attribute', c, data=geodataframe):
+    rgb = matplotlib.colors.to_rgba_array(c)
+    alpha_channel = x/y.max()
```

Comment:
This is great! could we generalize it a bit? I'd like to be able to have:
1. diverging scaling (extremes are colored, middle isnt, something like `(x-np.median(y)).abs() / np.abs(y).max()`
2. logarithmic scaling (including log divergent), which changes over the log of `y` (or `y + y.min()` rather than `y` directly, if `y` has negative elements)

Reply:
I can try add more functionality to #28 :)